system/bluetooth_monitor/service/l2ping.cpp

```diff
@@ -0,0 +1,364 @@
+// Copyright 2022 Autoware Foundation
```

Comment:
https://autowarefoundation.github.io/autoware-documentation/main/contributing/license/#license

```suggestion
// Copyright 2022 The Autoware Contributors
```

Reply:
Fixed in https://github.com/autowarefoundation/autoware.universe/pull/862/commits/442ef82559aba18b5b402dff982a2eac803fc314 and https://github.com/autowarefoundation/autoware.universe/pull/862/commits/ba11df3ea73b39554d918e6da3adcbc7a81a73cf